components/stylesheets/category.css

```diff
@@ -0,0 +1,120 @@
+@import '../../index.css';
+
+
+.category__heading {
+    width: 100%;
+    margin: auto;
+    text-align: center;
+    padding: 1rem 0;
+}
+
+.category__heading i {
+    background-color: transparent;
+    font-size: 2.5rem;
+}
+
+.category__heading h1 {
+    color: var(--primary-color);
+    font-size: 2.5rem;
+    width: 100%;
+}
+
+.category__heading i {
+    color: var(--primary-color);
+    padding: 0.4rem;
+    border-radius: 50%;
+    cursor: pointer;
+}
+
+.category > *{
+    width: 100%;
+}
+
+.category__card{
+    border-top: 6px solid var(--primary-color);
+    width: 100%;
+    position: relative;
+    padding: 2rem;
+    box-shadow: 0px 6px 10px rgba(0, 0, 0, 0.25);
+}
+
+.category__one{
+    background: radial-gradient(#98e8f0, #3fbafe);
+    align-self: center;
+    margin: 1rem;
+}
+
+.category__two .category__card:nth-child(1){
+    background: radial-gradient(#90c485, #539E43);
+    margin-bottom: 2rem;
+}
+
+.category__two .category__card:nth-child(2){
+    background: radial-gradient(#f1e19e, #FFDA3E);
+}
+
+.category__three{
+    background: radial-gradient(#fc90a8, #DD0031);
+    align-self: center;
+    margin: 1rem;
```

Comment:
Instead of giving margin seperately, you could have taken them in one utility class. like `class="category category_one"`

Reply:
margin is different for different cards it is not constant
that's why I avoided that